cmd/plural/crypto.go

```diff
@@ -371,8 +372,14 @@ func handleRecover(c *cli.Context) error {
 	if !ok {
 		return fmt.Errorf("could not find `key` in console-conf secret")
 	}
+	keyValue := string(key)
```

Comment:
Wouldn't it be possible to reuse the AESKey struct and related functions here?

```go
type AESKey struct {
	Key string
}
```

They're in `pkg/crypto/key.go`.

Reply:
good point let me check this

Reply:
works fine, thanks